cli/src/commands/workspaces/delete.ts

```diff
@@ -24,12 +24,15 @@ export default class WorkspacesDelete extends BaseCommand {
 
     const xata = await this.getXataClient();
 
-    const { confirm } = await prompts({
-      type: 'confirm',
-      name: 'confirm',
-      message: `Are you sure you want to delete workspace ${workspace}?`,
-      initial: true
-    });
+    const { confirm } = await this.prompt(
+      {
+        type: 'confirm',
+        name: 'confirm',
+        message: `Are you sure you want to delete workspace ${workspace}?`,
```

Comment:
```suggestion
        message: `Are you sure you want to delete the ${workspace} workspace?`,
```

For consistency with the rest of deletion messages